### ISSUE TITLE
NEW ODK AGGREGATE SERVER

### ISSUE THREAD

**Issue by ethan92429** [AguaClara/POST#31]
Hi @AguaClara/post-cornell I decided to create two new instances that run the ODK aggregate server in Google App Engine... one for production and one for development. It occurred to me that the way we had previously been hosting the code was very hard to follow... it involved installing docker on an Amazon Ec2 instance, connecting it to a MySQL server (preferably running on a different instance but ours wasn't) and most importantly it required an Amazon account that we are about to run out of free time on. Hence I took the liberty of switching to GAE, which is comparatively easy to maintain, and ODK aggregate natively supports it. I updated the POST passwords/certs with all the info. I haven't actually switched anything over... At first I'm just trying to copy the instance we had been hosting on Amazon. Please feel free to play with the GAE development server all you want! the URL is: http://post-aguaclara-development.appspot.com and the proposed production server is at http://post-aguaclara.appspot.com . Look in the passwords doc for more info. Lastly, please post here if you have any questions or suggestions! Thanks.

**Comment by jacuerp** [comment]
@ethan92429 Can you explain the Developer Console mix-up issue we talked about this morning? I want to fix the problem but I'm not sure exactly what the problem is or how to access the new credentials

thanks!

**Comment by ethan92429** [comment]
I deleted the project that had the side effect of providing an api channel  to the fusion tables. Log into the console and look at the projects. I added one named fusion table api or something like that. Now look at the credentials of that project. That's where you'll find the api key that you need to add to the post.visualizations application to see it on the website. But I already changed it so no worries, just look at my latest commit to see what happened.

**Comment by ethan92429** [comment]
@AguaClara/post-cornell we need to be careful about what we upload to production! I found a TEST form on there today. Those belong on development! Please respond here or ask me if you have q's about the difference between the production and development server. (Hint, http://post-aguaclara.appspot.com/ is production server! be careful with that one. Only upload fully tested and approved forms there)

**Comment by asm278** [comment]
Sorry, thats my bad. I wasn't convinced that collect should redirect because it didn't find any forms on the server. So I put that form on the server, and sure enough, collect has two separate preferences for the URL it queries. I didn't put a finalized form because we don't have a  finalized 1.5 and I meant to clean up right after. Sorry about that.

**Comment by ethan92429** [comment]
Seems like everyone is testing on the new development server which is fantastic! Once the finalized form is uploaded to the production site I'll be making sure that all the plants are pointed to the new production server.
